features/lorawan/lorastack/mac/LoRaMacClassBInterface.h

```diff
@@ -0,0 +1,167 @@
+#ifndef MBED_LORAWAN_LORAMACCLASSB_INTERFACE_H_
+#define MBED_LORAWAN_LORAMACCLASSB_INTERFACE_H_
```

Comment:
please move these 2 lines after the license header file

Reply:
Done!